crates/next-dev/tests/integration/next/image/basic/pages/index.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import img from "../public/triangle-black.png";
+import { useEffect } from "react";
+
+import { Deferred } from "@turbo/pack-test-harness/deferred";
+
+let testResult = new Deferred();
+
+export default function Home() {
+  useEffect(() => {
+    // Only run on client
+    import("@turbo/pack-test-harness").then(runTests);
+  });
+
+  return [
+    <Image
+      id="imported"
+      alt="test imported image"
+      src={img}
+      width="100"
+      height="100"
+    />,
+    <Image
+      id="local"
+      alt="test src image"
+      src="/triangle-black.png"
+      width="100"
+      height="100"
+    />,
+  ];
+}
+
+globalThis.waitForTests = function () {
+  return testResult.promise;
+};
+
```

Comment:
```suggestion
```

We don't need that when we import the harness with a normal import.

Reply:
There's a couple ways we can improve this (eg, using a test render), but I want to defer that to a PR that specific for testing.